src/platform/user/tests/profile/utilities/index.unit.spec.js

```diff
@@ -166,7 +161,7 @@ describe('Profile utilities', () => {
         },
       });
 
-      expect(mappedData.status).to.equal('SERVER_ERROR');
+      expect(mappedData.veteranStatus.status).to.equal('SERVER_ERROR');
```

Comment:
This one should not change either. And honestly these tests should probably be updated to make sure that we aren't mis-mapping data from the API response to the Redux store.